misc/bump/README.md

```diff
@@ -0,0 +1,126 @@
+required.
+
+[preprocess]: https://github.com/seL4/ci-actions/tree/master/preprocess
+
+## When is a bump required?
+
+There are typically two situations in which a bump is required:
+
+1. A [seL4][] pull request requires a *bump* for a change to seL4 which affects
+   the verified configurations, but which is believed not to require a proof
+   update.
+
+2. There is a proof update for an seL4 patch (usually with corresponding seL4
+   pull request), and we want to merge both the seL4 change and the proof update.
+
+## Bump requests from seL4 pull requests
+
+Some changes to verified configurations do not require proof updates. This
+includes most changes to boot code, and superficial changes which can be
+absorbed by `ccorres_rewrite`. In these cases, the preprocess test on the seL4
+pull request will fail, but the `diff` it shows in its output contains only
+proof-irrelevant changes. Usually someone on the seL4 pull request will ask for
+help from someone with verification experience.
+
+An experienced proof engineer should inspect the `diff` in the [preprocess][]
+logs for the test failure, and make a judgement about whether a proof update
+will be required. If unsure, you can run a full proof [testboard][] on the PR.
+But unless the PR is a release candidate, there is some room for error. You
+should balance the desire for quick turnaround on kernel updates against the
+desire to avoid broken verification regression and unplanned proof updates.
+
+Note that changes to boot code can cause `SimplExportAndRefine` to fail,
+particularly if the changes involve complex loops, or complex memory access
+patterns (combinations of array access with either constant index offsets or
+struct fields). In those cases, it might be worth obtaining an opinion from
+someone with `SimplExportAndRefine` experience. Otherwise, don't worry about it
+until it fails.
+
+If you think no proof update is required, approve the seL4 pull request to be
+merged. As soon as possible after it is merged, follow the bump script recipe
+below. If you think a proof update is required, or if you bump and regression
+subsequently fails, then flag the need for a verification patch.
+
+## Verification patches
+
+A proof update for an seL4 update requires merging pull requests for both `l4v`
+and `seL4`, and a manifest bump. The proof engineer who prepared the patch
+should already have made sure the seL4 pull request is rebased and tested the
+proof update against the rebased version of the seL4.
+
+It doesn't hurt to double check that the manifest displayed in the proof test
+log contains the commit hashes of the branches you want to merge, but if you are
+sure that everything is up to date, you can skip this step.
+
+Once you're happy that both PRs have been tested and reviewed appropriately,
+both can be merged. The seL4 PR should be merged first, followed without too
+much delay by the l4v PR. As soon as both PRs are merged, follow the bump script
```

Comment:
the "not too much delay" might not be clear to people, but is obvious in retrospect when explained

Reply:
oh, it is explained in the paragraph below. I guess people might go through my experience reading it, but no harm done.

Reply:
Yes, I it is a bit confusing. I might add a `(see below)` to it to increase patience in the reader ;-)